fern/components/CodeConsole/codeData.ts

```diff
@@ -0,0 +1,425 @@
+import alchemyGetAssetTransfersEthereumRequest from "./code-samples/alchemy_getAssetTransfers/ethereum-request";
```

Comment:
Unfortunately dynamic imports don't seem to work in Fern, requiring this ugliness